src/javascripts/background.js

```diff
@@ -11,6 +11,9 @@ import { LS } from './shared.js';
 import { defaults } from './shared.js';
 import { constants } from './shared.js';
 
+// Keep the service worker active
```

Comment:
Could we link to something future maintainers world find helpful?

Reply:
We could, though I hope this will only be a temporary fix. https://stackoverflow.com/questions/66618136/persistent-service-worker-in-chrome-extension has good info but isn't very official. https://developer.chrome.com/docs/extensions/develop/concepts/service-workers might be a good official starting point.

Reply:
Just to step back:  I appreciate that you included a comment in your PR.  Very helpful!  :smile:  I'm coming from the perspective that future maintainers may not read all the context of our discussions on GitHub and may become confused or misunderstand.
 
Perhaps we should call out that this is temporary?   And really, I think anything you've mentioned works (including not changing the comment).

Reply:
Thanks - I've tweaked the comment.